components/ome-xml/src/ome/units/UNITS.java

```diff
@@ -66,7 +66,7 @@ public String getName()
   public static final ome.units.unit.Unit<ElectricPotential> VOLT    = ome.units.unit.Unit.<ElectricPotential>CreateBaseUnit("SI.VOLT", "V");
   public static final ome.units.unit.Unit<Frequency>         HERTZ   = ome.units.unit.Unit.<Frequency>CreateBaseUnit("SI.HERTZ", "Hz");
   public static final ome.units.unit.Unit<Length>            METRE   = ome.units.unit.Unit.<Length>CreateBaseUnit("SI.METRE", "m");
-  public static final ome.units.unit.Unit<Length>            INCH    = ome.units.unit.Unit.<Length>CreateBaseUnit("Imperial.INCH", "in");
+  public static final ome.units.unit.Unit<Length>            INCH    = METRE.multiply(0.0254).setSymbol("in");
```

Comment:
OK, that makes sense.  I think I'm still misunderstanding something though, as it doesn't seem terribly difficult to make metric/imperial conversion work just for Length.  If ```isConvertible``` is overridden to allow conversion between systems, and ```convertValue``` is overridden to:

1. perform a conversion in the original measurement system, to get either the metre value (if the original system is metric) or the inch value (if the original system is imperial).
2. perform the metre <--> inch conversion as defined in this PR, and change measurement systems
3. perform a conversion within the new measurement system to get the desired output unit

then I would think the inter-system conversion would work with minimal effort.

I guess that's being slightly nitpicky though - what is in the PR works, we just lose the ability to distinguish between the actual measurement systems for length units.  If no one else feels strongly that we need that, happy to merge as-is.